tests/storages_tests/journal_tests/test_journal.py

```diff
@@ -220,3 +223,33 @@ def append_logs(self, logs: list[dict[str, Any]]) -> None:
 
     with pytest.warns(FutureWarning):
         _ = _CustomJournalBackendInheritingDeprecatedClass()
+
+
+@pytest.mark.parametrize(
+    "lock_obj", (DeprecatedJournalFileOpenLock, DeprecatedJournalFileSymlinkLock)
+)
+def test_future_warning_of_deprecated_file_lock_obj_paths(
+    lock_obj: type[DeprecatedJournalFileOpenLock | DeprecatedJournalFileSymlinkLock],
+) -> None:
+    with pytest.warns(FutureWarning):
+        dummy_file_path = "dummy"
+        lock_obj(filepath=dummy_file_path)
```

Comment:
How about using the `tmp_path` fixture?

```suggestion
def test_future_warning_of_deprecated_file_lock_obj_paths(
    tmp_path: pathlib.PurePath,
    lock_obj: type[DeprecatedJournalFileOpenLock | DeprecatedJournalFileSymlinkLock],
) -> None:
    with pytest.warns(FutureWarning):
        lock_obj(filepath=tmp_path)
```